components/Form.js

```diff
@@ -27,36 +33,35 @@ export default function Form({ onClose, onSubmit }) {
   async function handleSubmit(event) {
     event.preventDefault();
 
+    showModal.toggle("submit");
+
     if (!formData.image.startsWith("https://images.unsplash.com")) {
       setValidationError("Only images from unsplash.com are allowed");
       return;
     }
 
-    const response = await fetch("/api/games", {
-      method: "POST",
+    const url = showModal.modal.isEdit
+      ? `/api/games/${showModal.modal.game._id}`
+      : "/api/games";
+
+    const response = await fetch(url, {
+      method: showModal.modal.isEdit ? "PATCH" : "POST",
       headers: { "Content-Type": "application/json" },
       body: JSON.stringify(formData),
```

Comment:
you have both logic for create and update a game now inside the Form. A cleaner approach would be to have an "onSubmit" function as a prop and call it here. By this, you can pass two different functions into this Form Component, a "handleCreate" function when it acts as the addForm and a "handleUpdate" when its the editForm.